content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,512 @@
+
+```shell
+curl https://raw.githubusercontent.com/cri-o/packaging/main/get > crio-install
+```
+
+Run the installer script:
+
+```shell
+sudo bash crio-install
+```
+
+Enable and start the `crio` service:
+
+```shell
+sudo systemctl daemon-reload
+sudo systemctl enable --now crio.service
+```
+
+Quick test:
+
+```shell
+sudo systemctl is-active crio.service
+```
+
+The output is similar to:
+
+```
+active
+```
+
+Detailed service check:
+
+```shell
+sudo journalctl -f -u crio.service
+```
+
+### Install  network plugins
+
+The `cri-o` installer installs and configures the `cni-plugins` package. You can
+verify the installation running the following command:
+
+
+```shell
+/opt/cni/bin/bridge --version
+```
+
+The output is similar to:
+
+```
+CNI bridge plugin v1.5.1
+CNI protocol versions supported: 0.1.0, 0.2.0, 0.3.0, 0.3.1, 0.4.0, 1.0.0
+```
+
+To check the default configuration:
+
+```shell
+cat /etc/cni/net.d/11-crio-ipv4-bridge.conflist
+```
+
+The output is similar to:
+
+```json
+{
+  "cniVersion": "1.0.0",
+  "name": "crio",
+  "plugins": [
+    {
+      "type": "bridge",
+      "bridge": "cni0",
+      "isGateway": true,
+      "ipMasq": true,
+      "hairpinMode": true,
+      "ipam": {
+        "type": "host-local",
+        "routes": [
+            { "dst": "0.0.0.0/0" }
+        ],
+        "ranges": [
+            [{ "subnet": "10.85.0.0/16" }]
+        ]
+      }
+    }
+  ]
+}
+```
+
+{{< note >}}
+Make sure that the default `subnet` range (`10.85.0.0/16`) does not overlap with
+one of your active networks. If there is an overlap, you can edit the file and change it
+accordingly. Restart the service after the change.
+{{< /note >}}
+
+### Download and set up the kubelet
+
+Download the [latest stable release](/releases/download/) of the Kubelet.
+
+{{< tabs name="download_kubelet" >}}
+{{< tab name="x86-64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubelet"
+{{< /tab >}}
+{{< tab name="ARM64" codelang="bash" >}}
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/arm64/kubelet"
+{{< /tab >}}
+{{< /tabs >}}
+
+Configure:
+
+```shell
+sudo mkdir -p /etc/kubernetes/manifests
+```
+
+```shell
+sudo tee /etc/kubernetes/kubelet.yaml <<EOF
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+authentication:
+  webhook:
+    enabled: false # Do NOT use in production clusters!
+authorization:
+  mode: AlwaysAllow # Do NOT use in production clusters!
+enableServer: false
+logging:
+  format: text
+address: 127.0.0.1 # Restrict access to localhost
+readOnlyPort: 10255 # Do NOT use in production clusters!
+staticPodPath: /etc/kubernetes/manifests
+containerRuntimeEndpoint: unix:///var/run/crio/crio.sock
+EOF
+```
+
+{{< note >}}
+Because you are not setting up a production cluster, you are using plain HTTP
+(`readOnlyPort: 10255`) for unauthenticated queries to the kubelet's API.
+{{< /note >}}
```

Comment:
```suggestion

The authentication webhook is disabled and authorization mode is set to `AlwaysAllow` for the purpose of this tutorial. You can learn more about [Authorization modes](https://kubernetes.io/docs/reference/access-authn-authz/authorization/#authorization-modules) and [webhook authentication](https://kubernetes.io/docs/reference/access-authn-authz/webhook/) to properly configure kubelet in standalone mode in your environment.
{{< /note >}}
```

Reply:
I actually not sure if there are anything else can be used in standalone mode. Never looked deep there, but I feel that those settings might be OK even for prod.

Reply:
A tweak to the suggestion: use a site-relative URL

Reply:
Done.
@sftim, aren't those links site-relative URLs? (/docs/...)